decidim-meetings/app/cells/decidim/meetings/redesigned_meeting/meeting_minutes.erb

```diff
@@ -0,0 +1,3 @@
+<div class="editor-content">
+  <%= decidim_sanitize_editor translated_attribute meeting.closing_report %>
```

Comment:
There should be a title for all meaningful sections on the page. I think the meeting minutes would be considered as such.

Reply:
Not necessary, this content will fit inside the horizontal tabs, therefore no title is required

Reply:
I think all these sections should have a meaningful title for them to identify separate sections clearly on the page.

Reply:
![imagen](https://user-images.githubusercontent.com/817526/213141903-ef57953e-5380-4c5f-bd54-a8913c363e93.png)

The meaningful title is given by the tab trigger. This is about the content, so append another title inside would be duplication.

In any case, please do not share design-matter concerns in a PR review, do it in a separate issue.